requirements.txt

```diff
@@ -46,4 +46,4 @@ sqlparse==0.5.0
 google-api-python-client==2.88.0
 sentry-usage-accountant==0.0.11
 freezegun==1.2.2
-sentry-protos==0.1.40
+sentry-protos @ git+https://github.com/getsentry/sentry-protos@pierre/eap-find-traces#subdirectory=py
```

Comment:
I assume this will be bumped once getsentry/sentry-protos#71 is merged + released?